projects/website/src/plots/PlotDriver.py

```diff
@@ -0,0 +1,323 @@
+                            elif key_1.lower() == "X2UpperBound".lower():
+                                job.X2UpperBound = float(value_1)
+                                continue 
+                            elif key_1.lower() == "Y1LowerBound".lower():
+                                job.Y1LowerBound = float(value_1)
+                                continue 
+                            elif key_1.lower() == "Y2LowerBound".lower():
+                                job.Y2LowerBound = float(value_1)
+                                continue 
+                            elif key_1.lower() == "Y1UpperBound".lower():
+                                job.Y1UpperBound = float(value_1)
+                                continue 
+                            elif key_1.lower() == "Y2UpperBound".lower():
+                                job.Y2UpperBound = float(value_1)
+                                continue 
+                            elif key_1.lower() == "X1Label".lower():
+                                job.X1Label = value_1
+                                continue 
+                            elif key_1.lower() == "Y1Label".lower():
+                                job.Y1Label = value_1
+                                continue 
+                            elif key_1.lower() == "X2Label".lower():
+                                job.X2Label = value_1
+                                continue 
+                            elif key_1.lower() == "Y2Label".lower():
+                                job.Y2Label = value_1
+                                continue 
+                            elif key_1.lower() == "ExperimentalData".lower():
+                                job.experimentalData = value_1
+                                continue 
+                            elif key_1.lower() == "DataFileOverride".lower():
+                                job.dataFile = value_1
+                                continue 
+                            elif key_1.lower() == "DataPathOverride".lower():
+                                job.dataPath = value_1
+                                continue 
+                            elif key_1.lower() == "PFTFile".lower():
+                                job.PFTFile = value_1
+                                continue 
+                            elif key_1.lower() == "OutputOverride".lower():
+                                job.outputDir = value_1
+                                continue 
+                            elif key_1.lower() == "FontSize".lower():
+                                job.fontSize = int(value_1)
+                                continue 
+                            else:
+                                logging.warning("Unrecognized config directive: " + directive)   
+            fin.close()
+        except Exception as e:
+            logging.error("Ouch Something went wrong")
+
+    def execute(self):
+        for job in self.jobs:
```

Comment:
I'm sure python has a simple thread pool so you can queue all these jobs in a word queue and give it to the threads then wait for the pool to close.  

https://www.tutorialspoint.com/concurrency_in_python/concurrency_in_python_pool_of_threads.htm

We can talk about it tomorrow but where possible if we could thread these scripts it would be useful.